developer-docs-site/docs/reference/move.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Move on Aptos"
+slug: "move"
+sidebar_position: 0
+---
+
+# Move on Aptos
+
+The Aptos blockchain consists of validator nodes that run a consensus protocol. The consensus protocol agrees upon the ordering of transactions and their output when executed on the Move Virtual Machine (MoveVM). Each validator node translates transactions along with the current blockchain ledger state as input into the VM. The MoveMV processes this input to produce a changeset or storage delta as output. Once consensus agrees and commits to the output, it becomes publicly visible. In this guide, we will introduce you to core Move concepts and how they apply to developing on Aptos.
+
+## What is Move?
+
+Move is a safe and secure programming language for Web3 that emphasizes **scarcity** and **access control**. An assets in Move can be represented by or stored within *resource*. **Scarcity** is enforced by default as structs cannot be duplicated. Only structs that have explicitly been defined at the bytecode layer as *copy* can be duplicated.
```

Comment:
1. One thing about move I really like is the resource /data is stored under my account. compared to ETH, the data is stored under the contract account.

Reply:
Look at the slides, I agree, there's some nuance here.